contracts/ColonyTask.sol

```diff
@@ -484,7 +484,20 @@ contract ColonyTask is ColonyStorage {
 
     colonyNetworkContract.appendReputationUpdateLog(role.user, reputation, domains[task.domainId].skillId);
     if (taskRole == TaskRole.Worker) {
-      colonyNetworkContract.appendReputationUpdateLog(role.user, reputation, task.skills[0]);
+      int256 nSkills = 0;
```

Comment:
Noting that if we're removing skill penalties we should be able to treat these at `uint`s now.

Reply:
Maybe I've misunderstood you, but we're only removing the penalty on the global skill given to the worker for not rating the manager. They might still have a negative reputation change overall if they've been rated -1, so this needs to remain `int` or I need to cast a lot in the following lines. I opted for the former.